### PR TITLE
Issue 541 custom kakao click support

### PR DESCRIPTION
**Pull request by matzuk**
The full support of Kakao clicks in Kaspresso is done in this PR.

### CODE REVIEW

File: docs/Wiki/Kaspresso_configuration.en.md [hunk 1]
```diff
@@ -22,6 +22,25 @@ class SomeTest : TestCase(
 
 **Kaspresso** configuration contains:
 
+#### Kakao clicks
+Kaspresso provides the possibility to override custom Espresso clicks by custom clicks.
```
Comment (Nikitae57) [inline docs/Wiki/Kaspresso_configuration.en.md:26]: Override custom clicks by custom clicks... Maybe just "override espresso custom clicks"?

File: docs/Wiki/Kaspresso_configuration.en.md [hunk 2]
```diff
@@ -22,6 +22,25 @@ class SomeTest : TestCase(
...
+Kakao library provides a set of prepared custom clicks which improves the stability of tests especially on devices under the high load.
```
Comment (Nikitae57) [inline docs/Wiki/Kaspresso_configuration.en.md:27]: Suggest "the" before "tests" and "devices", and "a" before "high load"

File: docs/Wiki/Kaspresso_configuration.ru.md [hunk 3]
```diff
@@ -22,6 +22,25 @@ class SomeTest : TestCase(
 
 Конфигурация **Kaspresso** содержит:
 
+#### Kakao clicks
+Kaspresso предоставляет возможность переопределить стандартные клики Espresso.
```
Comment (Nikitae57) [inline docs/Wiki/Kaspresso_configuration.ru.md:26]: Maybe "Кастомные клики"  to align with an English doc

File: docs/Wiki/Kaspresso_configuration.ru.md [hunk 4]
```diff
@@ -22,6 +22,25 @@ class SomeTest : TestCase(
...
+Kakao библиотека предоставляет набор подготовленных имплементаций кликов, которые улучшают стабильность тестов на девайсах, находящихся под большой нагрузкой.
+
+Все детали о проблеме и решениях описано в [Kakao документации](https://github.com/KakaoCup/Kakao/tree/master/kakao-ext-clicks).
+
+Пример, как заиспользовать кастомные клики в вашем тесте представлено в [ClickTest](../../samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/click_tests/ClickTest.kt).
```
Comment (Nikitae57) [inline docs/Wiki/Kaspresso_configuration.ru.md:31]: "," after "тесте", "предоставлен" (without o on the end)

File: samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/click_tests/ClickTest.kt [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+import com.kaspersky.kaspresso.kaspresso.Kaspresso
+import com.kaspersky.kaspresso.params.ClickParams
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import org.junit.Rule
+import org.junit.Test
+
+/**
+ * The example of how to apply kakao custom clicks in Kaspresso
+ */
+class ClickTest : TestCase(
```
Comment (Nikitae57) [inline samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/click_tests/ClickTest.kt:20]: Suggest "CustomClickTest". We have plenty of clicks in the other tests

**Review by Nikitae57 — CHANGES_REQUESTED** [review]

**Comment by AzamatCherchesov** [comment]
Some problems with detekt, @matzuk have a look pls

**Review by Nikitae57 — APPROVED** [review]

**Review by AzamatCherchesov — APPROVED** [review]

**Pull request merged**